### ISSUE TITLE
Issue of layer name parsing (self-defined layer) and weight converting

### ISSUE THREAD

**Issue by ChunfeiMa** [nerox8664/pytorch2keras#26]
I got a model which has some layers as following: 

State dict: ['base.conv1.weight', 'base.bn1.weight', 'base.bn1.bias', 'base.bn1.running_mean', 'base.bn1.running_var', 'base.layer1.0.conv1.weight', ..., 'local_conv_list.0.0.weight', 'local_conv_list.0.0.bias', 'local_conv_list.0.1.weight', 'local_conv_list.0.1.bias', 'local_conv_list.0.1.running_mean', ..., 'fc_list.0.weight', 'fc_list.0.bias', 'fc_list.1.weight', 'fc_list.1.bias', 'fc_list.2.weight', 'fc_list.2.bias', 'fc_list.3.weight', 'fc_list.3.bias', 'fc_list.4.weight', 'fc_list.4.bias', 'fc_list.5.weight', 'fc_list.5.bias']

The definition of the model is : ResNet50 + self.local_conv_list = nn.ModuleList() 
where: 
self.local_conv_list.append(nn.Sequential(
        nn.Conv2d(2048, local_conv_out_channels, 1),
        nn.BatchNorm2d(local_conv_out_channels),
        nn.ReLU(inplace=True)
      ))

In converter.py 

        node_weights_name = '.'.join(
            re.findall(r'\[([\w\d.]+)\]', node_scope_name)
        )

this function parsing the string, such like: 
['PCBModel/ResNet[base]/Sequential[layer4]/Bottleneck[2]/Conv2d[conv3]']
into the original pytorch layer name  such as :
base.layer4.2.conv3
then converting the weights accordingly.

But when it comes to the layer which does not have certain naming pattern, such like:
'local_conv_list.0.1.weight',
graph node: PCBModel/Sequential/Conv2d[0]
name in state_dict: 0

the parsing results will be something wrong and it will never matched with any layer name in the State dict, therefore results in the converting error.

So, I got stuck here and was wondering how to handle it, is there anyone could give some constructive suggestion?
